### ISSUE TITLE
Abnormal proportion of CD4-positive, alpha-beta TEMRA T cells

### ISSUE THREAD

**Issue by BenSolomon** [obophenotype/human-phenotype-ontology#11024]
I would recommend a term structure for CD4 TEMRA cells that is analogous to CD8 TEMRA cells

**Preferred term label:**

Abnormal proportion of CD4-positive, alpha-beta TEMRA T cells
Increased proportion of CD4-positive, alpha-beta TEMRA T cells
Decreased proportion of CD4-positive, alpha-beta TEMRA T cells

**Synonyms**

**Definition (free text, please give PubMed ID)**
An abnormal proportion of CD4-positive, alpha-beta effector memory RA TEMRA T cells compared to the total number of T cells in the blood. These cells have the phenotype CD45RA-positive, CD45RO-negative, and CCR7-negative.

Relevant references
- PMID 29133794
- PMID 34860194

**Parent term (use hpo.jax.org/app)**
I recommend the following structure:

├── HP:0410379 Abnormal proportion of CD4-positive, alpha-beta memory T cells 
│   ├── Abnormal proportion of CD4-positive, alpha-beta TEMRA T cells
│   │   ├── Increased proportion of CD4-positive, alpha-beta TEMRA T cells
│   │   ├── Decreased proportion of CD4-positive, alpha-beta TEMRA T cells

This would mirror the existing structure for CD8 TEMRA cells:

├── HP:0410380 Abnormal proportion of CD4-positive, alpha-beta memory T cells 
│   ├── HP:0020177 Abnormal proportion of CD8-positive, alpha-beta TEMRA T cells
│   │   ├── HP:0500265 Increased proportion of CD8-positive, alpha-beta TEMRA T cells
│   │   ├── HP:0500266 Decreased proportion of CD8-positive, alpha-beta TEMRA T cells

Related to HP:0020177 - Abnormal proportion of CD8-positive, alpha-beta TEMRA T cells. Just as there can be CD8+ TEMRA cells, there can be CD4+ TEMRA cells. 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 603122 - Supported by PMID 26083206. Patients with DOCK2 deficiency have abnormalities in both CD4+ TEMRA cells and CD8+ TEMRA cells.

**Your nano-attribution (ORCID)**
0000-0003-1009-7312

**Comment by pnrobinson** [comment]
@adamklocperk
thoughts?

**Comment by adamklocperk** [comment]
@pnrobinson @BenSolomon It would be acceptable to have a mirror structure for CD4 and CD8 T cell memory terms. The reason why Kimberly, Catharina, Beata and I decided not to do so is that in CD4 T cells, the TEMRA population is poorly pronounced.

I also note that Increased CD4+ memory T cell proportion HP:0410391 has weird children, see here:
[Decreased CD4+ effector memory T cell proportion](https://hpo.jax.org/browse/term/HP:0025624)
[Increased CD4+ effector memory T cell proportion](https://hpo.jax.org/browse/term/HP:0025625)
[Increased proportion of central memory CD4-positive T cells](https://hpo.jax.org/browse/term/HP:0410393)
[Increased proportion of effector memory CD4-positive T cells](https://hpo.jax.org/browse/term/HP:0410394)

HP:0410394 and HP:0025625 are synonyms.
HP:0025624 should be a child of Decreased CD4+ memory T cell proportion HP:0410386.

If you decide that CD4 TEMRAs should be included, I would keep the terminology identical, e.g. Increased/Decreased CD4+ TEMRA T cell proportion.

**Comment by pnrobinson** [comment]
@adamklocperk 
We implemented these corrections:
HP:0410394 and HP:0025625 are synonyms.
HP:0025624 should be a child of Decreased CD4+ memory T cell proportion HP:0410386.

**Comment by pnrobinson** [comment]
@adamklocperk We are finding a lot of articles about CD4+ TEMRA cells, e.g., https://www.nature.com/articles/s41467-017-01728-5
@BenSolomon 

We do not understand "the TEMRA population is poorly pronounced." - does this mean it is harder to measure them? Should we add a comment?

**Comment by BenSolomon** [comment]
Thanks for the comments everyone! 

@pnrobinson I'm sure @adamklocperk will follow up, but at least from my clinical perspective, CD4+ TEMRA cells are routinely reported in our T-cell flow cytometry. Any flow cytometry capable of recognizing CD8+ memory populations will include CD4, CD8, CD45RA/RO, and CCR7 +/- CD62L, so in addition to identifying the CD8 TEMRAs (CD8+ CD45RA+ CCR7-), will also identify the CD4 TEMRAs (CD4+ CD45RA+ CCR7-).

However, in the basic immunology literature, CD8+ TEMRAs are what originally defined the concept of a TEMRA cell and are much more well profiled than the CD4+ TEMRA, which is what I believe @adamklocperk was referring to. However, thinking about the patient curation that is done by ClinGen, for example, case reports that include T-cell phenotyping by flow cytometry as a table, will often include CD4+ TEMRAs even if the significance of the cells isn't fully appreciated. So in that sense, being able to curate the CD4+ TEMRAs with a corresponding term could provide researchers/clinicians with an opportunity to identify some patterns across studies that could shed some more light on the significance of these cells relative to the CD8 TEMRAs.

**Comment by pnrobinson** [comment]
@aslgraefe Let's address this one too -- what was the feedback?

**Comment by aslgraefe** [comment]
@pnrobinson @BenSolomon Thank you for this!
We discussed the CD4+ TEMRA cells in the last ESID effort and decided to add "[increased CD4+ TEMRA cells](https://hpo.jax.org/browse/term/HP:5210377)" since they were explicitly mentioned in this [PMID:34860194](https://pubmed.ncbi.nlm.nih.gov/34860194/), but in this and the other PMID mentioned in the issue text above, I could not find explicitly decreased CD4+ TEMRA cells being mentioned. But in case I missed the part in those PMIDs, please let me know. Therefore, we also did not add the term "abnormal CD4+ TEMRA" since I only found PMID evidence for the increased term. 
If there are cases reported in ClinGen with "decreased CD4 TEMRA cells", then I guess we could add this and the "abnormal" umbrella term, same as for the [cd8+ temra branch](https://hpo.jax.org/browse/term/HP:0020177)

**Comment by adamklocperk** [comment]
Coming back to this, I think increased CD4 TEMRAs is a reasonable term, but would agree that decreased CD4 TEMRAs is a hard to justify, as these are usually extremely low to almost nonexistent outside of pathological situations - this is unlike CD8 TEMRAs, which are quite well represented in the healthy population.

**Comment by pnrobinson** [comment]
@BenSolomon @aslgraefe for there to be an HPO term, there would need to be a lower limit of normal where we could say that an individual has an abnormally low value. Since that is not the case, this is not a valid HPO term. Thanks!
